### PR TITLE
Create Blog “2022-07-15-opencilk-2-0-released”

### PR DESCRIPTION
**Pull request by behoppe**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by behoppe** [comment]
Thanks @neboat. Sounds good. I understand that I am a final reviewer. I have two accounts and just removed my personal @beh333 from the OpenCilk organization as a bit of house-cleaning, but now I can't assign myself to review my own pull request. But I understand that I am the reviewer for this PR.

File: src/posts/2022-07-15-opencilk-2-0-released.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: OpenCilk 2.0 released
+tagline: OpenCilk 2.0 features a new syntax and implementation for reducer
+  hyperobjects, an upgraded compiler, improved support for pedigrees, and more.
+author: Tao B. Schardl
+date: 2022-07-20T19:08:51.844Z
+tags:
+  - news
+  - release
+---
+OpenCilk 2.0, which features several changes and enhancements over OpenCilk 1.1, is now available.  See the [Install](/doc/users-guide/install) page for instructions on how to download and install OpenCilk 2.0.
```
Comment (beh333) [inline src/posts/2022-07-15-opencilk-2-0-released.md:12]: I suggest streamlining in the following way:
```suggestion
OpenCilk 2.0 is now available.  See the [Install](/doc/users-guide/install) page for instructions on how to download and install.
```

File: src/posts/2022-07-15-opencilk-2-0-released.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Major changes
+
+OpenCilk 2.0 features the following major changes from OpenCilk 1.1:
+- **[Beta]** Cilk reducer hyperobjects (a.k.a., reducers) are now supported through a new language syntax and implementation.  A local or global variable in C or C++ can be made into a reducer by adding `cilk_reducer(I,R)` to its type, where `I` and `R` designate the identity and reduce functions for the reducer.  See the [Reducer documentation](/doc/reference/reducers) for details.
+
+- The compiler has been upgraded to be based on LLVM 14.0.6.
+- Support for pedigrees and built-in deterministic parallel random-number generation has been improved and optimized.  In particular, pedigrees are now correctly updated at both spawns and syncs.
```
Comment (beh333) [inline src/posts/2022-07-15-opencilk-2-0-released.md:20]: Clarifying the subject-verb agreement:
```suggestion
- Support has been improved and optimized for pedigrees and built-in deterministic parallel random-number generation. In particular, pedigrees are now correctly updated at both spawns and syncs.

```

**Review by beh333 — APPROVED** [review]
Looks good! I made two small changes for grammar/style.

**Pull request merged**